src/main/java/uk/gov/justice/hmpps/prison/repository/sql/InmateAlertRepositorySql.kt

```diff
@@ -10,6 +10,7 @@ enum class InmateAlertRepositorySql(val sql: String) {
         ALERT_CODE,
         COALESCE(alCode.DESCRIPTION, ALERT_CODE) as ALERT_CODE_DESC,
         EXPIRY_DATE,
+        MODIFY_DATETIME,
```

Comment:
this column in ambiguous and needs the `OA.` prefix - `STAFF_USER_ACCOUNTS` also has a column called `MODIFY_DATETIME`